### PR TITLE
Implement Verify Message - Closes #2407

### LINKED ISSUE #2407: Verify message
### Description 
We have a feature which enables users to sign a given text using their account and create hash. This is then verifiable using the public Key.
The usage of this feature is to prove ownership of the account without having to send tokens.
At the moment we don't have a page in Lisk Hub to verify the hashed message. We had this is Lisk Nano.

This is the "sign message" page: form and result. and the verify message should look like this:

Please find the designs in [InVision](https://projects.invisionapp.com/d/main#/projects/prototypes/18253439).

### Acceptance criteria
 - Inputs can't be empty (invalid).
 - If users switch to different tabs the content of the inactive tab won't be cleared out.
 - The public key must be a valid pbk.
 - There's no validation on the single input form except it must not be empty.

### PR DESCRIPTION
**Pull request by slaweet**
### What issue have I solved?

Fixes #2407

### How have I implemented/fixed it?

Implemented the new page as defined in the [designs](https://projects.invisionapp.com/d/main#/console/18253439/381870437/inspect)

Note that the public key input error feedback in the designs has already the new design, but this has to be changed across all pages that use error feedback in a separate PR. 

### How has this been tested?

- Go to https://ci.lisk.io/test/lisk-hub/PR-2434/#/verify-message?message=Hello%20world&publicKey=c094ebee7ec0c50ebee32918655e089f6e1a604b83bcaa760293c61e0f18ab6f&signature=c68adc13131696c35ac82b9bb6884ee4de66ff281b013fe4ded66a73243c860b6a74b759bfb8d25db507ea2bec4bb208f8bb514fa18380416e637db947f0ab06
- Verify that message 
- Or go to https://ci.lisk.io/test/lisk-hub/PR-2434/#/sign-message and create a signed message and try to verify it
- Or try to verify this message:

```
-----BEGIN LISK SIGNED MESSAGE-----
-----MESSAGE-----
Hello world
-----PUBLIC KEY-----
c094ebee7ec0c50ebee32918655e089f6e1a604b83bcaa760293c61e0f18ab6f
-----SIGNATURE-----
c68adc13131696c35ac82b9bb6884ee4de66ff281b013fe4ded66a73243c860b6a74b759bfb8d25db507ea2bec4bb208f8bb514fa18380416e637db947f0ab06
-----END LISK SIGNED MESSAGE-----
```

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

File: src/components/screens/verify-message/verifyMessageInput.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+            <Tooltip className="showOnBottom">
+              <p>{t('If you would like to ensure the ownership of another account, you can send a message via Lisk Hub and request the recipient to sign it.')}</p>
+            </Tooltip>
+          </Box.InfoText>
+          <div className={styles.inputViewSwitcher}>
+            {t('Input view')}
+            <Icon
+              className="inputs-view-icon"
+              name={`verifyMessageInputsView${!isInputsView ? '' : 'Active'}`}
+              onClick={this.changeView.bind(this, true)}
```
Comment (osvaldovega) [inline src/components/screens/verify-message/verifyMessageInput.js]: is better to bind this in the constructor and not in the render

File: src/components/screens/verify-message/verifyMessageInput.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+            />
+            <Icon
+              className="textarea-view-icon"
+              name={`verifyMessageTextareaView${isInputsView ? '' : 'Active'}`}
+              onClick={this.changeView.bind(this, false)}
```
Comment (osvaldovega) [inline src/components/screens/verify-message/verifyMessageInput.js]: same in here

File: src/components/toolbox/box/infoText.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+@import '../../app/variables.css';
+@import '../../app/mixins.css';
+
+.infoText {
+  @mixin contentLargest;
+
+  letter-spacing: 0.21px;
+  line-height: 25px;
```
Comment (osvaldovega) [inline src/components/toolbox/box/infoText.css]: do you need to set these in here, I think that the purpose of the mixin is for not to do it.
I am talking about letter-spacing and line-height

**Comment by sridharmeganathan** [comment]
I reviewed this story and found two minor issues.

1. Message text box is accepting empty space.
2. Message text box is accepting empty space before & after text.

**Comment by slaweet** [comment]
@sridharmeganathan thank you for the feedback.
What is the expected behavior?
- should there be an error message under the message input? If so what should it say?
- should the "Continue" button be disabled if Massage is empty or has the extra whitespace?

**Comment by reyraa** [comment]
> should there be an error message under the message input? If so what should it say?
yes. according to criteria _Inputs can't be empty (invalid)_. and the message can simply be _The message can't be empty._

> should the "Continue" button be disabled if Massage is empty or has the extra whitespace?
yes.

Thank you

**Comment by slaweet** [comment]
@sridharmeganathan I implemented the changes according to https://github.com/LiskHQ/lisk-hub/pull/2434#issuecomment-533042986 please check again.

**Comment by slaweet** [comment]
@sridharmeganathan I fixed the leading/trailing whitespace condition. Please check again.

**Pull request merged**